_platforms/en/io.md

```diff
@@ -0,0 +1,39 @@
+---
+title: IO
+subtitle: The app for public services
+logo: /assets/images/logo-io@2x.png
+payoff: The app for public services
+description: IO is the application of the Italian government that allows all citizens to use national and local public services from their smartphone in a simple, modern and secure way.
+
+lang: en
+order: 4
+highlight: true
+comingsoon: false
+layout: platform
+github_team: io
+---
+
+## Intro
+
+IO is the application of the Italian government that allows all citizens to use national and local public services from their smartphone in a simple, modern and secure way.
+It is an Open Source project that is growing every day involving different entities and services.
+
+IO aims to facilitate the relationship between citizens and the Public Administration (PA) through the creation of a platform of reusable components capable of making digital services more effective.
+
+The application's components can be reused by administrations that provide digital services in order to ensure a consistent user experience as well as save on the effort required by a new implementation of these technologies.
+
+The goal is to provide a simpler and more familiar way for the Public Administration to communicate with citizens, notify deeds, remember deadlines, share information and documents through an interface built on the citizen needs.
+
+## Preferences management API and communication API
+
+The preferences management API allow citizens to manage all the information associated with their digital citizen profile. Thanks to this information (communication channels, language preference, privacy settings, etc…) it is possible to create personalized digital services.
+
+The communication APIs provide a single channel that the Public Administration can use to send communications, notices and deadlines to the citizen. This communication system, together with the preferences management API, allows services to communicate with citizens on the basis of his own preferences.
+
+### How to contribute
+
+* [API Usage](https://teamdigitale.github.io/digital-citizenship/)
+* [GitHub Repository](https://github.com/teamdigitale/digital-citizenship-functions)
+* [How to contribute](https://github.com/teamdigitale/digital-citizenship/blob/master/CONTRIBUTING.it.md)
```

Comment:
```suggestion
* [API Usage](https://developer.io.italia.it/openapi.html)
* [GitHub Repository](https://github.com/pagopa/io-app)
* [How to contribute](https://github.com/pagopa/io/blob/master/CONTRIBUTING.it.md)
```

Reply:
The links were fixed by #748 for the Italian version.

Reply:
fixed :+1: